test/start.js

```diff
@@ -433,3 +402,62 @@ test('should start the server listening on 0.0.0.0 when runing in docker', t =>
     t.strictEqual(fastify.server.address().address, '0.0.0.0')
   })
 })
+
+test('should start the server with watch options that the child process restart when directory changed', t => {
+  t.plan(5)
+
+  const tmpjs = path.resolve(baseFilename + '.js')
+  fs.writeFileSync(tmpjs, 'hello world')
+
+  setTimeout(function () {
+    const argv = [ '-p', '3000', '-w', './examples/plugin.js' ]
+    const fastifyEmitter = start.start(argv)
+
+    t.tearDown(() => {
+      if (fs.existsSync(tmpjs)) {
+        fs.unlinkSync(tmpjs)
+      }
+      fastifyEmitter.emit('close')
+    })
+
+    fastifyEmitter.on('start', () => {
+      t.pass('should receive start event')
+      setTimeout(function () {
+        t.pass('touch tmpjs')
+        // fastifyEmitter.emit('ready')
+        // fastifyEmitter.emit('restart')
+        exec(`touch ${tmpjs}`) // chokidar watch can't caught change event in travis CI, but local test is all ok. you can remove annotation in local environment.
```

Comment:
Can you avoid using bash? I would prefer the tests to be able to run on Windows as well.